src/main/CommandHandler.cpp

```diff
@@ -1124,16 +1130,27 @@ CommandHandler::clearMetrics(std::string const& params, std::string& retStr)
 }
 
 void
-CommandHandler::surveyTopology(std::string const& params, std::string& retStr)
+CommandHandler::checkBooted() const
 {
-    ZoneScoped;
-
     if (mApp.getState() == Application::APP_CREATED_STATE ||
         mApp.getHerder().getState() == Herder::HERDER_BOOTING_STATE)
     {
         throw std::runtime_error(
             "Application is not fully booted, try again later");
     }
+}
+
+void
+CommandHandler::surveyTopology(std::string const& params, std::string& retStr)
```

Comment:
I added a deprecation warning to `stopSurvey`. `getSurveyResults` remains in the new interface. I updated the example in `admin.md`, so that should give an idea of what HTTP endpoints I added or kept in the new survey interface and how to use them. I also added the new endpoints to `commands.md`, as well as notes about which endpoints are deprecated.